calculator/operations.py

```diff
@@ -1,23 +1,25 @@
-
 def add(*args):
-    # your implementation here
-    pass
+    return(sum(args))
 
 def subtract(*args):
-    # your implementation here
-    pass
+    ans = float(args[0])
```

Comment:
You don't need to cast your subtraction/multiplication stuff as a float initially. You'll get the same value leaving it as it is. It's only an issue when you're trying to divide in python 2.x as then it defaults to integer division. (eg. it will return 3/2 = 1) For addition/subtraction/multiplication floats will only come into play when one of the numbers you're passing in is already a float, in which case python will handle it properly anyways. (eg. if multiply received 1.3 and 2 as arguments even without casting it would return 2.6)